blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -111,54 +41,45 @@ export function PresentationalNav(props) {
 		primaryLogoAlt,
 	} = props;
 	return (
-		<StyledNav
+		<Stack
 			id="main-nav"
-			className={navColorClass}
-			navBarBackground={backgroundColor}
-			navHeight={navHeight}
-			scrolled={scrolled}
-			// hard-coded to medium-breakpoint
-			breakpoint={mediumBreakpoint}
+			className={BLOCK_CLASS_NAME}
 			aria-label={sectionAriaLabel}
+			direction="horizontal"
+			alignment="center"
```

Comment:
if you have to change alignment based on a breakpoint, the stack can't be used. you'd need to use a a plain <nav> and modify based on the breakpoint